### PR TITLE
Update Documentation (Fixed typos).

### PR DESCRIPTION
**Pull request by NinjaCoderDev**

### CODE REVIEW

**Comment by botberry** [comment]
Hi, thanks for contributing to Strawberry 🍓!

We noticed that this PR is missing a `RELEASE.md` file. We use that to automatically do releases here on GitHub and, most importantly, to PyPI!

So as soon as this PR is merged, a release will be made 🚀.

Here's an example of `RELEASE.md`:

```markdown
Release type: patch

Description of the changes, ideally with some examples, if adding a new feature.
```

Release type can be one of patch, minor or major. We use [semver](https://semver.org/),so make sure to pick the appropriate type. If in doubt feel free to ask :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 First off, thanks for taking the time to contribute!
 
 The following is a set of guidelines for contributing to Strawberry on GitHub.
-These are mostly guidelines, not rules. Use your best judgement, and feel free
```
Comment (patrick91) [inline CONTRIBUTING.md]: this works in both forms ;)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -49,19 +48,16 @@ the problem:
 - **Describe the exact steps which reproduce the problem** in as many details as
   possible.
 - **Provide specific examples to demonstrate the steps to reproduce the issue**.
-  Include links to files or GitHub projects, or copy-paste-able snippets, which
-  you use in those examples.
+  Include links to files or GitHub projects, or copy-paste-able snippets, which you use in those examples.
 - **Describe the behavior you observed after following the steps** and point out
-  what exactly is the problem with that behavior.
+  what exactly is the problem with that behavior?
```
Comment (patrick91) [inline CONTRIBUTING.md]: this was fine like it was before, it is not a question 😊

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -99,15 +95,15 @@ where you can create a new one and provide the following information:
...
+- **Provide specific examples to demonstrate the steps**...
 - **Describe the current behavior** and **explain which behavior you expected to
   see instead** and why.
 
 ### Contributing to code
 
 #### Local development
 
-You will need Poetry to start contributing on the Strawberry. Refer to the
+You will need Poetry to start contributing to the Strawberry. Refer to the
```
Comment (patrick91) [inline CONTRIBUTING.md]:
```suggestion
You will need Poetry to start contributing to Strawberry. Refer to the
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -139,7 +134,7 @@ is in order:
 $ poetry run pre-commit install
 ```
 
-Your code must always be accompanied by corresponding tests, if tests are not
+Your code must always be accompanied by corresponding tests if tests are not
```
Comment (patrick91) [inline CONTRIBUTING.md]:
```suggestion
Your code must always be accompanied by corresponding tests, if tests are not
```

the comma was needed :)

File: README.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 # Strawberry GraphQL
 
-> Python GraphQL library based on dataclasses
+> Python GraphQL library based on data classes
```
Comment (patrick91) [inline README.md]:
```suggestion
> Python GraphQL library based on dataclasses
```

Here we are referring to the dataclasses module: https://docs.python.org/3/library/dataclasses.html

File: README.md [hunk 6]
```diff
@@ -89,7 +89,7 @@ urlpatterns = [
 ## Contributing
 
 We use [poetry](https://github.com/sdispater/poetry) to manage dependencies, to
-get started follow these steps:
+get started to follow these steps:
```
Comment (patrick91) [inline README.md]:
```suggestion
get started follow these steps:
```

was correct already 😊

File: README.md [hunk 7]
```diff
@@ -116,7 +116,7 @@ pre-commit install
 - Repository: https://github.com/strawberry-graphql/strawberry
 - Issue tracker: https://github.com/strawberry-graphql/strawberry/issues
   - In case of sensitive bugs like security vulnerabilities, please contact
-    patrick.arminio@gmail.com directly instead of using issue tracker. We value
+    patrick.arminio@gmail.com directly instead of using the issue tracker. We value
```
Comment (patrick91) [inline README.md]: 👍

File: RELEASE.md [hunk 8]
```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+Fixed some typos in readme.md and contributing.md .
```
Comment (patrick91) [inline RELEASE.md]:
```suggestion
Fixed some typos in readme.md and contributing.md.
```

**Comment by NinjaCoderDev** [comment]
@patrick91 Thanks for the opportunity , I have resolved the conflicts and where can I send you email .
Thanks again.

**Comment by patrick91** [comment]
@NinjaCoderDev my email is in my github profile ;)

**Pull request merged**